src/main/java/cmput402/tdd/Recipe.java

```diff
@@ -20,39 +20,39 @@ public void setName(String name){
         this.name = name;
     }
 
-    public void add(Item item, Integer quantity) throws RuntimeException{
+    public void add(Item item, Integer quantity) throws Exception{
         if(items.get(item) == null && quantity > 0){
             items.put(item, quantity);
         }
         else if(items.get(item) != null && quantity > 0){
             items.put(item, items.get(item)+quantity);
         }
         else{
-            throw new RuntimeException("Error: item cannot be added");
+            throw new Exception("item cannot be added");
```

Comment:
Error message should be capitalized and should tell the user why it can not be added

Reply:
Changed to more meaningful commit message